_aggregations/metric/maximum.md

```diff
@@ -9,30 +9,46 @@ redirect_from:
 
 # Maximum aggregations
 
-The `max` metric is a single-value metric aggregations that returns the maximum value of a field.
+The `max` metric is a single-value metric that returns the maximum value of a field.
 
-The following example calculates the maximum of the `taxful_total_price` field:
+The `max` aggregation compares numeric fields using a `double` (double-precision) representation. Results should be considered approximate for fields containing `long` or `unsigned_long` integer values greater than 2^53, since the number of significant bits in a `double` mantissa is 53.
```

Comment:
A little bit of an edge case, but bears mentioning. Please evaluate for accuracy and ease of understanding.